app/views/collections/_form.html.slim

```diff
@@ -20,7 +20,7 @@
               = f.nested_fields_for :collection_tasks do |ct|
                 - task = ct.object.task
                 - if task.present?
-                  .row[data-id=ct.object.task_id]
+                  .row[data-id=ct.object.task.id]
```

Comment:
Sorry for not being clear - Since you already assigned task right above, I thought it would be nice to use it.
```suggestion
                  .row[data-id=task.id]
```